docker-compose.yaml

```diff
@@ -46,7 +46,7 @@ services:
       - ./backend:/var/www/html
     labels:
       - 'traefik.backend=nginx'
-      - 'traefik.port=80'
+      - 'traefik.port=${TRAEFIK_NGINX_PORT:-80}'
```

Comment:
This change is unnecessary, Traefik is the only service that is exposed on port 80, and Traefik re-routes it's port to the port defined for the relevant services.

So Nginx can still run internally on port 80 while Traefik runs on port 8080.